main.py

```diff
@@ -629,27 +718,20 @@ def handle_link_liveness_disabled(self, interfaces) -> None:
         for link in links.values():
             self.notify_link_status_change(link, reason="liveness_disabled")
 
-    @listen_to("kytos/.*.link_available_tags")
+    @listen_to("kytos/core.link_available_tags")
```

Comment:
On the kytos PR I suggested to refactor this a new event, since actually it's per interface instead of link now, so the old name isn't appropriate anymore. If you could double check it.

Reply:
Changed name to `kytos/core.interface_tags`